pathod/language/actions.py

```diff
@@ -52,7 +50,7 @@ def intermediate(self, settings):  # pragma: no cover
 
 
 class PauseAt(_Action):
-    unique_name = None
+    unique_name = None  # type: ignore
```

Comment:
I have done that before , but was getting some errors like
`error: Definition of "unique_name" in base class "_HeaderMixin" is incompatible with definition in base class "Token"
`
Then from [here](https://github.com/python/mypy/issues/1237) I get to know that either we should refactor code or go for `# type: ignore`